csharp/ql/src/Security Features/CWE-134/UncontrolledFormatString.ql

```diff
@@ -31,7 +31,13 @@ class FormatStringConfiguration extends TaintTracking::Configuration {
   }
 }
 
+string getSourceType(DataFlow::Node node) {
+  result = node.(RemoteFlowSource).getSourceType()
+  or
+  result = node.(LocalFlowSource).getSourceType()
+}
+
 from FormatStringConfiguration config, DataFlow::PathNode source, DataFlow::PathNode sink
 where config.hasFlowPath(source, sink)
-select sink.getNode(), source, sink, "$@ flows to here and is used as a format string.",
-  source.getNode(), source.getNode().toString()
+select sink.getNode(), source, sink, "Format string depends on a $@.", source.getNode(),
+  ("this" + getSourceType(source.getNode()))
```

Comment:
Looks wrong?

Reply:
Yep, there is an `a` too much.  
I've removed the `a` such that the alert-message is similar to the one you use in `cs/sql-injection`.